layouts/_default/baseof.html

```diff
@@ -24,43 +24,16 @@
     {{ partial "footer/footer.html" . }}
     {{ partial "footer/script-footer.html" . }}
 
-    <!-- Google Tag Manager -->
-    <script type="text/plain" data-cookiecategory="analytics">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-TR443T9');</script>
-    <!-- End Google Tag Manager -->
-
-    <!-- Send Custom Event to Google Analytics for the live search functionality. -->
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-HCQWPQKQH8"></script>
```

Comment:
Should the id be different in each documentation? If yes can we add it as a parameter or partial?

Reply:
?
```suggestion
    <script async src="https://www.googletagmanager.com/gtag/js?id={{ .Site.Params.tagManagerId }}"></script>
```